jfleet-core/src/test/java/org/jfleet/mysql/MySqlReservedWordTest.java

```diff
@@ -29,12 +30,15 @@
 import javax.persistence.Table;
 
 import org.jfleet.BulkInsert;
+import org.jfleet.parameterized.DatabaseArgumentProvider;
 import org.jfleet.util.MySqlDatabase;
 import org.jfleet.util.SqlUtil;
 import org.junit.jupiter.api.Test;
 
 public class MySqlReservedWordTest {
 
+    private final MySqlDatabase database = (MySqlDatabase) DatabaseArgumentProvider.getDatabaseContainer(MySql);
```

Comment:
To get a database instance I don't like:
* To call to a `DatabaseArgumentProvider` class that is related with Junit parameterized tests
* To cast to a type that doesn't provide value

To solve this, I propose you:
* in tests `org.jfleet.util` package create a `DatabaseProvider` class and move `getDatabaseContainer` method to this class
* Rename the method, because... we are not talking about containers, just databases :)
* User of this method wants just a `Database` implementation, but doesn't need to have it explicitly defined. Can we just assign it to a `Database` variable type?

As conclusion, to have something like this:

```java
private final Database database = DatabaseProvider.getDatabase(MySql);
```

Reply:
It's much cleaner that way :) I'ts already done